tooling/nargo_fmt/tests/expected/expr.nr

```diff
@@ -85,10 +85,11 @@ fn parenthesized() {
 }
 
 fn parenthesized() {
-    value + (
-        // line
-        x as Field
-    )
+    value
+        + (
+            // line
+            x as Field
+        )
```

Comment:
The previous diff looks better than this, not sure why it has put  ` + (` on a separate line